src/fmu/tools/rms/set_data_constant.py

```diff
@@ -0,0 +1,196 @@
+def set_data_constant(config: dict):
+    """Set data from RMS constant.
+
+    This method is a utility in order to set surface and 3D grid property data
+    to a given value. The value must be of the correct type (if discrete 3D
+    property for example). The purpose of it is to make sure that those data
+    are properly generated by the modelling workflow and not inherited from a
+    previous run with the corresponding jobs deactivated. The data are set to a
+    value triggering attention and not deleted in order not to reset some jobs
+    in RMS.
+
+    The input of this method is a Python dictionary with defined keys. The keys
+    "project" and "value" are required while "horizons", "zones" and
+    "grid_models" are optional (at least one of them should be provided for the
+    method to have any effect).
+
+    Args:
+        project: The roxar magic keyword ``project`` refering to the current
+            RMS project.
+
+        value: The constant value to assign to the data. It could be 0 or -999
+            for example. If discrete properties from grid models are modified,
+            the value should be applicable (integer).
+
+
+        horizons: A Python dictionary where each key corresponds to the name of
+            the horizons category where horizon data need to be made empty. The
+            value associated to this key should be a list of horizon names to
+            modify. If a string ``all`` is assigned instead of a list, all
+            available horizon names for this category will be used.
+            Alternatively, if a list of horizons categories is given instead of
+            a dictionary, the method will apply to all horizons within these
+            horizons categories.
+
+        zones: A Python dictionary where each key corresponds to the name of
+            the zones category where zone data need to be made empty. The value
+            associated to this key should be a list of zone names to modify. If
+            a string ``all`` is assigned instead of a list, all available zone
+            names for this category will be used.
+            Alternatively, if a list of zones categories is given instead of a
+            dictionary, the method will apply to all zones within these zones
+            categories.
+
+        grid_models: A Python dictionary where each key corresponds to the name
+            of the grid models where properties need to be made empty. The
+            value associated to this key should be a list of property names to
+            modify. If a string ``all`` is assigned instead of a list, all
+            available properties for this grid model name will be used.
+            Alternatively, if a list of grid models names is given instead of a
+            dictionary, the method will apply to all properties within these
+            grid models.
+    """
+
+    import roxar
```

Comment:
Why is this import needed?

Reply:
We use lines like: `grid = project.grid_models[gridname]` where project is recognized as RMS project structure only if roxar library is on.

Reply:
Would it work to put it into a try/except clause to make the linting happy?
e.g.:
```
try:
    import roxar
except ImportError:
    pass
```